package.json

```diff
@@ -4,18 +4,14 @@
   "description": "jquery.inputmask is a jquery plugin which create an input mask.",
   "main": "./dist/inputmask/jquery.inputmask.js",
   "files": [
-    "dist/inputmask/jquery.inputmask.js",
-    "dist/inputmask/inputmask.js",
-    "dist/inputmask/inputmask.extensions.js",
     "dist/inputmask/inputmask.date.extensions.js",
```

Comment:
npm automatically alphabetizes lists in package.json.

I also included the whole `extra/phone-codes/` directory. The reason that the js-files are listed explicitly is that minified is in the same directory